lib/app.js

```diff
@@ -102,15 +101,35 @@ class App {
       if      (shell.test('-e', oldTrufflePath)) truffleConfig = reqCwd.silent(oldTrufflePath);
       else if (shell.test('-e', newTrufflePath)) truffleConfig = reqCwd.silent(newTrufflePath);
```

Comment:
@vreturn one last thing - believe we could also resolve #359 in this PR if we reverse the order of the config type tests here - e.g. test for newTrufflePath first because that's what Truffle prefers when both configs are present.